src/main.js

```diff
@@ -193,3 +195,17 @@ function resetTimerView() {
   timerButton.classList.remove('meditate-button');
   timerButton.classList.remove('exercise-button');
 }
+
+function localStorageActivity() {
+  if (localStorage) {
+    for (var i = 0; i < localStorage.length; i++) {
+      var activityID = localStorage.key(i);
+      var activityObject = JSON.parse(localStorage.getItem(activityID));
```

Comment:
What a great function!! It's saying if there is local storage then for loop local storage array and assign a variable to the key at index of i, next, assign a variable to parseing the object and getting the key of the varaible